src/atoms/Switch/Switch.native.js

```diff
@@ -219,7 +219,7 @@ const Switch = ({ disabled, on, defaultOn, onChange, size, testID }) => {
       animatedRightValue.setValue(animationConfig.on.rightSpace);
       setToggle(true);
     }
-  }, [on, defaultOn]);
+  }, [on, defaultOn, toggle, animatedLeftValue, animatedRightValue, animationConfig]);
```

Comment:
Why aren't these things caught earlier as warning?

Reply:
We have this [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks), right?

Reply:
Yes! 2 possible reasons. Either the lint was bypassed or the plugin was added later 🤦‍♂